searx/engines/google.py

```diff
@@ -112,9 +112,9 @@
 # specific xpath variables
 # ------------------------
 
-results_xpath = '//div[contains(@class, "MjjYud")]'
-title_xpath = './/h3[1]'
-href_xpath = './/a/@href'
+results_xpath = './/div[@data-sokoban-container]'
```

Comment:
Cool, seems more robust then what we have tried in the [past (select by ``contains(@class, "..."`` )](https://github.com/searxng/searxng/commits/master/searx/engines/google.py)